backend/resolvers/types/userType.js

```diff
@@ -1,7 +1,18 @@
 export default {
   User: {
     // Judges (and Admins) have shows
```

Comment:
I'm confused about the `(and Admins)` part of this comment and how it relates to the rest of this function.

If any user, regardless of type, makes this query, they'll get the shows ordered by `'judgingStart` unless they provide a date, no? Remind me how Students and Admins are associated to shows – would they need a different default? 

Additionally, for Judges, it seems useful to know if you've been assigned to any future show, not just those that are in the submission or judging phase, so that you can plan to come back at a future date.

Reply:
No, because this hooks onto the user_show table, which is only used in assigning judges. There's no student show association formally. We could add a check for students and show them something else. Students and admins should be able to to vote because Nanette still wants to function as a judge. 

As far as what they see I specifically asked about the range. I can open it to any future show if you believe that's better

Reply:
Ah, ok. 

Technically, though, if a relation was made between a Student and a Show (like manually editing the db), calling this query would return that row, right? I'm just trying to understand how architecturally things fit together – it just feels weird that in the schema there are functions which are only relevant to one type of user but not another – I'm wondering if there's a better schema design (i.e. for that type of relation – probably not worth the time / effort to change things / re-engineer now, but I'm just thinking about future GraphQL projects).

For Students, might make sense to do an empty object or "Permission Denied" since they're not allowed to vote.

I'm sorry if I mis-understood what you were asking about at our meeting about the date range. When I was testing this out yesterday, I had to make new shows in the future, and I was a bit confused that even after I assigned judges to the shows, I couldn't view them as a judge. UX-wise I think showing all future shows the judges have been assigned to is a better workflow – say Nanette emails them that they're supposed to vote on an upcoming show, but that show hasn't entered the submission phase yet, might confuse judges who log on and says they haven't been assigned to any shows.

Reply:
You're right. I'll open the date range. I can also make students always get an empty array.

Reply:
Slightly related. we probably should beef up our seed to make it a little easier to test.